BackEnd/Backend/Backend.Bl/Configuration/OrderCommandsConfiguration.cs

```diff
@@ -0,0 +1,6 @@
+namespace Backend.Bl.Configuration;
+
+public class OrderCommandsConfiguration
```

Comment:
Add it to the configuration json and to DI (AddConfiguration function in DependencyInjectionExtensions)